services/121-service/src/payments/fsp-integration/intersolve-visa/mappers/intersolve-visa-status.mapper.ts

```diff
@@ -0,0 +1,98 @@
+import { VisaCardAction } from '@121-service/src/payments/fsp-integration/intersolve-visa/enum/intersolve-visa-card-action.enum';
+import { IntersolveVisaCardStatus } from '@121-service/src/payments/fsp-integration/intersolve-visa/enum/intersolve-visa-card-status.enum';
+import { IntersolveVisaTokenStatus } from '@121-service/src/payments/fsp-integration/intersolve-visa/enum/intersolve-visa-token-status.enum';
+import { VisaCard121Status } from '@121-service/src/payments/fsp-integration/intersolve-visa/enum/wallet-status-121.enum';
+import csvParser from 'csv-parser';
+import fs from 'fs';
+import * as path from 'path';
+
+const StatusUnknownExplain = `Status is unknown, please contact the 121 Platform Team`;
+
+interface VisaCard121StatusMapInterface {
+  TokenBlocked: boolean;
+  TokenStatus: string;
+  CardStatus: string;
+  VisaCard121Status: string;
+  VisaCard121StatusExplanation: string;
+  Actions121: string;
+}
+
+interface VisaCard121StatusInformationAndActions {
+  status: VisaCard121Status;
+  explanation: string;
+  actions: VisaCardAction[];
+}
+
+const VisaCardActionsMapping = {
+  Pause: VisaCardAction.pause,
+  Unpause: VisaCardAction.unpause,
+  Reissue: VisaCardAction.reissue,
+};
+
+export class IntersolveVisaStatusMapper {
+  private static mapping: VisaCard121StatusMapInterface[] = [];
+
+  constructor() {
+    IntersolveVisaStatusMapper.loadMapping();
```

Comment:
If you use a static method or property of a class, the constructor of that class will not be executed automatically, so the code in this constructor is never called I think. I also ran into this when I was also working on this, which is why I put this loadMapping in the main.ts with a TODO comment. 

So if you remove `IntersolveVisaStatusMapper.loadMapping();` on line 70 I don't think the mapping is ever loaded.

A suboptimal solution I can think of is to load the mapping in the constructor of the intersolve visa service and than pass it along as parameter through the static functions to `determineVisaCard121StatusInformation`